portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,135 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
-
-#greeting-container {
-  margin-top: 20px;
-}
+body {
```

Comment:
It is good to organize the properties in a css rule in alphabetical order. I find that this helps to easily find specific property without scanning the entire rule. Please make that change everywhere in the file.

body {
  background-attachment:
  background-image:
  color:
  ...
}